aszczachor/src/views/mainPage/MainContent.js

```diff
@@ -41,22 +43,41 @@ const photos = [
   img18,
   img19,
   img20,
-  img21,
 ];
 
 const MainContent = () => {
+  const [selectedPhoto, setSelectedPhoto] = useState(null);
```

Comment:
to tak naprawdę `selectedPhotoIndex` co nie? zmieniłbym nazwę w takim wypadku, bo ta sugeruje url do obrazka